UMLClass.py

```diff
@@ -0,0 +1,111 @@
+'''
+Samantha Noggle
+
+Adds a class, deletes a class, & renames a class
+'''
+
+# List of all class objects the user has created
+classIndex = []
+
+class UMLClass:
+    def __init__(self, name: str):
+        self.name = name
+        self.attributes = []
+
+    def rename(self, newName):
```

Comment:
do we accept any name even if it is taken

Reply:
You would have to ask Sam about that because that's her code. I would imagine no, since the requirement is a unique name.

Reply:
In the constructor, yes. But in the addClass method, it checks before creating the class and adding it to the classIndex. The constructor would never really be used except by the addClass function. Two class objects with the same name will never be in the classIndex.